AkitaInteractive/Hachiko-Engine
# Added resource configuration menu & ability to have different wrapping modes and min/mag filters for textures in texture batching.

# How to Test:
- Download the "ugly cliff.zip" under the "Artists" folder in Akita Drive, and put the textures and the model into the engine.
- Load the model on a scene (empty or an existing)
- Change the settings by using the "... Texture Resource" dropdown on ComponentMaterial.
- Observe the changes
- Save the scene, close the engine, open again and see the texture changes applied
- Do another change on just the texture and do not save the scene and close the engine
- Open the engine and see the changes auto saved on texture only
